### ISSUE TITLE
How to forcibly unload and delete any created pipelines?

### ISSUE THREAD

**Issue by Meatfucker** [deepbeepmeep/mmgp#11]
Hi, im liking your module and the great work youve put into it. Im running into an issue where if I call the following pipeline function once, it works fine, but if I then call it again Im ooming. My attempt to manually delete the pipeline seems to have failed and I was hoping there was some way to force the module to clear things.

`import gc
import torch
from diffusers import LTXPipeline
from mmgp import offload, profile_type

async def generate_ltx(prompt,
                       input_video=None,
                       height=512,
                       width=704,
                       num_frames=161,
                       guidance_scale=3,
                       seed=None):

    generator = LTXPipeline.from_pretrained("Lightricks/LTX-Video", torch_dtype=torch.bfloat16).to("cpu")
    offload.profile(generator, profile_type.LowRAM_HighVRAM)
    
    video = generator(prompt=prompt,
                      width=width,
                      height=height,
                      num_frames=num_frames,
                      num_inference_steps=50).frames[0]

    offload.unload_all()

    del generator
    gc.collect()
    torch.cuda.empty_cache()

    return video
`
